.github/workflows/coding-standard-baseline.yml

```diff
@@ -0,0 +1,67 @@
+name: M2 Coding Testing
+
+on:
+  push:
+    branches: [ 2.4-develop, develop, codingstandard_baseline_integration ]
+  pull_request:
```

Comment:
I think we should use the same as here:

https://github.com/mage-os/mageos-magento2/blob/631315d7243593023330cfa93b9710f0422b2e68/.github/workflows/coding-standard.yml#L3-L6

Also, if we have `pull_request` we do not need `push` as well, right?

Reply:
Yes we added above for testing only